src/program.cpp

```diff
@@ -628,6 +630,13 @@ void program::from_value(const value& v)
     {
         MIGRAPHX_THROW("Warning: Program version mismatch");
     }
+    auto migx_version = v.at("migraphx_version").to<std::string>();
+    if(migx_version != get_migraphx_version())
+    {
+        std::cerr << "MIGraphX version mismatch, consider recompiling model with environment "
+                     "variable MIOPEN_FIND_ENFORCE=3 to re-tune the model "
```

Comment:
I dont think the `MIOPEN_FIND_ENFORCE=3` makes sense if the model is not using MIOpen. This should say `Warning: MIGraphX version mismatch. Operators may not be compatible`. The problem with different migraphx versions is incompatible operators. Its unrelated to MIOpen tuning.